### PR TITLE
LRQA-58148 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:relevant

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties [hunk 1]
```diff
@@ -117,7 +117,15 @@
     # https://bit.ly/2IkAwwW for Liferay Portal and https://bit.ly/2GIyfZF for
     # Liferay DXP.
     #
-    #liferay.workspace.target.platform.version=7.3.2
+#if (${liferayVersion.startsWith("7.0")})
+    liferay.workspace.target.platform.version=7.0.6
```
Comment (petershin) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties:121]: Does this line need to be commented out too?

Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties:121]: No, we have to roll back the liferay.workspace.product key for now. Greg is sending a new workspace that will fix that. Until then, we'll default back to using target platform as default. 
But Brian's asking that I send him the change directly

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by petershin** [comment]
Hi @lawrence-lee, in general I'll remove commented out code.  If `liferay.workspace.product` is not being used, the entire section should be removed until the code that's using it is re-enabled.

For `liferay.workspace.target.platform.version`, please set it only once with the value that's needed.  If its not used, the entire section should be removed too

**Comment by lawrence-lee** [comment]
@petershin. Got it. I'll send you a followup pull to clean that up.

**Pull request closed without merging**